tests/testsuite/cargotest/support/mod.rs

```diff
@@ -120,9 +119,9 @@ impl ProjectBuilder {
     }
 
     pub fn new(name: &str, root: PathBuf) -> ProjectBuilder {
+        drop(name);
```

Comment:
Can we just remove the `name` parameter if it's not used?  It doesn't look like it affects too many call sites.

Reply:
Sure thing!